lib/linux_admin/process/memory.rb

```diff
@@ -0,0 +1,49 @@
+module LinuxAdmin
+  module Process
+    class Memory
```

Comment:
Another option is to write a `Process#memory` method that just returns one of these Process::Memory instances.